server/pulp/server/db/manage.py

```diff
@@ -191,6 +193,14 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        # Prompt the user if there are active workers or running tasks
+        if status.get_workers() or filter(lambda x: x.state in constants.CALL_INCOMPLETE_STATES,
```

Comment:
Also what about checking the timestamps? This will only work if all processes are gracefully shut down. If we check the timestamps also we could pick up cases for non-graceful shutdown. What do you think?